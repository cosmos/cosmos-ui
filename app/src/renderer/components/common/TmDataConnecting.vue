<template>
  <tm-data-msg>
    <div slot="title">Waiting for connection</div>
    <div slot="subtitle">
      You are currently not connected to a node. Voyager will load this data
      when a connection is established.
    </div>
  </tm-data-msg>
</template>

<script>
import TmDataMsg from "common/TmDataMsg"
export default {
  name: `tm-data-connecting`,
  components: { TmDataMsg }
}
</script>
