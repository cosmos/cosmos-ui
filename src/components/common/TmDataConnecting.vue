<template>
  <div class="tm-data-msg">
    <img class="tm-data-msg__icon" src="~assets/images/loader.svg" />
    <div class="tm-data-msg__text">
      <div slot="title" class="tm-data-msg__title">
        Finding connection
      </div>
      <div slot="subtitle" class="tm-data-msg__subtitle">
        You are currently not connected to a node. We will load this information
        when a connection is established.
      </div>
    </div>
  </div>
</template>

<script>
export default {
  name: `tm-data-connecting`
}
</script>
