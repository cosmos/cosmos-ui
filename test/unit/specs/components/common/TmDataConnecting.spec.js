import { mount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import TmDataConnecting from "common/TmDataConnecting"

describe(`TmDataConnecting`, () => {
  let wrapper
  beforeEach(() => {
    wrapper = mount(TmDataConnecting)
  })

  it(`has the expected html structure`, () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
