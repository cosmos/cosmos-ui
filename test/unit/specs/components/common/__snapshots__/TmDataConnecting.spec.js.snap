// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmDataConnecting has the expected html structure 1`] = `
<div
  class="tm-data-msg"
>
  <div
    class="tm-data-msg__icon"
  >
    <i
      class="material-icons"
    >
      
      
    
    </i>
  </div>
   
  <div
    class="tm-data-msg__text"
  >
    <div
      class="tm-data-msg__title"
    >
      <div>
        
    Waiting for connection
  
      </div>
    </div>
     
    <div
      class="tm-data-msg__subtitle"
    >
      <div>
        
    You are currently not connected to a node. Voyager will load this data
    when a connection is established.
  
      </div>
    </div>
  </div>
</div>
`;
